SDK/MobileCenter/Microsoft.Azure.Mobile.UWP/Utils/AbstractScreenSizeProvider.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Mobile.Utils
+{
+    public abstract class AbstractScreenSizeProvider : IScreenSizeProvider
```

Comment:
I thought Abstract prefix was more Java and .NET was using `Base` suffix.
https://stackoverflow.com/a/826857/598520